Comparing source compatibility of /root/workspace/kl0u/flink/389213917/flink-streaming-java/target/flink-streaming-java_2.11-1.6-SNAPSHOT.jar against /root/./workspace/kl0u/flink/389213917/.m2/org/apache/flink/flink-streaming-java_2.11/1.5.0/flink-streaming-java_2.11-1.5.0.jar
***  MODIFIED CLASS: PUBLIC org.apache.flink.streaming.api.datastream.KeyedStream  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.datastream.KeyedStream withWatermarkAssigners(org.apache.flink.api.java.functions.SelectiveWatermarkAssigner[])
